CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+### Added
+- Support for SSL requests (@seanrobb)
```

Comment:
Ideally it would be best to describe this in greater detail such as:
```
### Added
- metrics-springboot.rb: new option `--protocol` to allow specifying https connection for requests  (@seanrobb)
- metrics-springboot.rb: new option `--url` to be used as a single parameter instead of specifying `--protocol`, `--host`, `--port`, and `--path` (@seanrobb)
```